common/common.go

```diff
@@ -7,19 +7,22 @@ import (
 	"strings"
 	"time"
 
-	"github.com/btcsuite/btcd/rpcclient"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 	"github.com/zcash-hackworks/lightwalletd/parser"
 	"github.com/zcash-hackworks/lightwalletd/walletrpc"
 )
 
-func GetSaplingInfo(rpcClient *rpcclient.Client, log *logrus.Entry) (int, int, string, string) {
+// Stubs for testing
+type RpcFunc func(method string, params []json.RawMessage) (json.RawMessage, error)
+type SleepFunc func(d time.Duration)
+
+func GetSaplingInfo(rf RpcFunc, sf SleepFunc, log *logrus.Entry) (int, int, string, string) {
```

Comment:
The `rf` (rpc function) and `sf` (sleep function) arguments allow unit tests to stub (mock) those functions; the reason to mock sleep is so that the unit test doesn't really sleep on retry.